src/main/webapp/script.js

```diff
@@ -332,3 +347,20 @@ function createCommentElement(name, message, time) {
   commentElement.innerText = name + ' posted ' + message + ' on ' + time;
   return commentElement;
 }
+
+/**
+ * Adds ID to form submission.
+ * @return {number} Current school's ID.
+ */
+function setReviewsButton() {
```

Comment:
Let's rename this function. You're not actually setting the reviews button, but setting the form action and school ID hidden input. I think if you named it something like `prepReviewsForm` or similar, that would be a bit more accurate.